### PR TITLE
Throw exception if property doesn't exist, fixed tests in windows.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | https://github.com/yiisoft/yii-form/issues/9

### CODE REVIEW

File: src/FormModel.php [hunk 1]
```diff
@@ -289,6 +290,11 @@ private function readProperty(string $attribute)
         $getter = fn ($class, $attribute) => $class->$attribute;
         $getter = \Closure::bind($getter, null, $this);
 
+        $class = get_class($this);
+        if (!property_exists($class, $attribute)) {
+            throw new InvalidArgumentException("Undefined property: \"$class $attribute\".");
```
Comment (rustamwin) [inline src/FormModel.php:295]:
```suggestion
            throw new InvalidArgumentException("Undefined property: \"$class::$attribute\".");
```

File: src/FormModel.php [hunk 2]
```diff
@@ -289,6 +290,11 @@ private function readProperty(string $attribute)
         $getter = fn ($class, $attribute) => $class->$attribute;
         $getter = \Closure::bind($getter, null, $this);
 
+        $class = get_class($this);
+        if (!property_exists($class, $attribute)) {
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply src/FormModel.php:294]: done.

File: src/FormModel.php (hunk 1 above)
Comment (terabytesoftw) [reply src/FormModel.php:295]: done.

File: tests/Widget/NoExistTest.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+final class NoExistTest extends TestCase
+{
+    public function testNoExistWidget(): void
+    {
+        $data = new LoginForm();
+
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('Undefined property: "Yiisoft\Form\Tests\Stub\LoginForm::undefined_form_attribute".');
+
+        (new TextInput)->config($data, 'login')->run();
```
Comment (xepozz) [inline tests/Widget/NoExistTest.php:22]: Why did you do this and next one line? 
It should be removed

File: tests/Widget/NoExistTest.php [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        (new PasswordInput)->config($data, 'password')->run();
+        (new TextInput)->config($data, 'undefined_form_attribute')->run();
```
Comment (xepozz) [inline tests/Widget/NoExistTest.php:24]:
```suggestion
        (new TextInput())->config($data, 'undefined_form_attribute')->run();
```

File: tests/Widget/NoExistTest.php (hunk 3 above)
Comment (terabytesoftw) [reply tests/Widget/NoExistTest.php:22]: fix

File: tests/Widget/NoExistTest.php (hunk 4 above)
Comment (terabytesoftw) [reply tests/Widget/NoExistTest.php]: fix

**Comment by samdark** [comment]
Thank you!

**Pull request merged**